django/db/models/functions/math.py

```diff
@@ -0,0 +1,121 @@
+from django.db.models import Func, Transform, fields
+
+
+class Abs(Transform):
+    function = 'ABS'
+    lookup_name = 'abs'
+
+
+class ACos(Transform):
+    function = 'ACOS'
+    lookup_name = 'acos'
+    output_field = fields.FloatField()
```

Comment:
Please import `FloatField` directly instead of the `fields` module.

Reply:
Good point. Updated already. Thanks.